README.md

```diff
@@ -16,33 +16,36 @@
 <!-- omit in toc -->
 ## Table of Contents
 
-- [Overview](#overview)
```

Comment:
My IDE "fixes" this... It has been like this for years and I never bothered to fix it... If it's not important, I'd appreciate it if we just ignore this.